src/base/bittorrent/torrenthandle.cpp

```diff
@@ -836,6 +836,22 @@ bool TorrentHandle::isInactive() const
     return !isActive();
 }
 
+bool TorrentHandle::isStalled() const
+{
+    return isStalledUp()
+            || isStalledDown();
+}
+
+bool TorrentHandle::isStalledUp() const
```

Comment:
Please don't use full words Downloading/Uploading instead of Down/Up.
Also shouldn't you use Seeding instead of Uploading to be consistent with other places?

Reply:
@glassez I used Up(loading) instead of Seeding for consistency with other places where other variables/functions/etc with related functionality have names like stalledUP (serialize_torrent) and StalledUploading (BitTorrent::TorrentState::StalledUploading).